demisto_sdk/commands/common/constants.py

```diff
@@ -1845,3 +1863,38 @@ class ParameterType(Enum):
 FORMATTING_SCRIPT = "indicator-format"
 
 ENV_SDK_WORKING_OFFLINE = "DEMISTO_SDK_OFFLINE_ENV"
+
+XSOAR_PREFIX_TAG = "<~XSOAR>\n"
+XSOAR_SUFFIX_TAG = "\n</~XSOAR>\n"
+XSOAR_INLINE_PREFIX_TAG = "<~XSOAR>"
+XSOAR_INLINE_SUFFIX_TAG = "</~XSOAR>"
+
+XSOAR_SAAS_PREFIX_TAG = "<~XSOAR_SAAS>\n"
+XSOAR_SAAS_SUFFIX_TAG = "\n</~XSOAR_SAAS>\n"
+XSOAR_SAAS_INLINE_PREFIX_TAG = "<~XSOAR_SAAS>"
+XSOAR_SAAS_INLINE_SUFFIX_TAG = "</~XSOAR_SAAS>"
+
+XSOAR_ON_PREM_PREFIX_TAG = "<~XSOAR_ON_PREM>\n"
+XSOAR_ON_PREM_SUFFIX_TAG = "\n</~XSOAR_ON_PREM>\n"
+XSOAR_ON_PREM_INLINE_PREFIX_TAG = "<~XSOAR_ON_PREM>"
+XSOAR_ON_PREM_INLINE_SUFFIX_TAG = "</~XSOAR_ON_PREM>"
+
+XSIAM_PREFIX_TAG = "<~XSIAM>\n"
+XSIAM_SUFFIX_TAG = "\n</~XSIAM>\n"
+XSIAM_INLINE_PREFIX_TAG = "<~XSIAM>"
+XSIAM_INLINE_SUFFIX_TAG = "</~XSIAM>"
+
+XPANSE_PREFIX_TAG = "<~XPANSE>\n"
+XPANSE_SUFFIX_TAG = "\n</~XPANSE>\n"
+XPANSE_INLINE_PREFIX_TAG = "<~XPANSE>"
+XPANSE_INLINE_SUFFIX_TAG = "</~XPANSE>"
```

Comment:
Why do you need this here?

Reply:
This constant is used in a few files. I think it is better for one source of truth to prevent typos and double declarations.